versions/1/integration/manifest.spec.yml

```diff
@@ -97,6 +97,10 @@ spec:
             type: string
             examples:
               - "image/svg+xml"
+          dark_mode:
```

Comment:
You may want to add it also to the input type package.

Reply:
It should already be done automatically. Input packages are using directly the same definition as integration for icons, screenshots, vars, etc. From [input spec](https://github.com/elastic/package-spec/blob/main/versions/1/input/manifest.spec.yml#L107): 
```yaml
    icons:
      $ref: "../integration/manifest.spec.yml#/definitions/icons"
    screenshots:
      $ref: "../integration/manifest.spec.yml#/definitions/screenshots"
```

Reply:
Awesome 👍